mlir/lib/Dialect/LLVMIR/IR/GENXOps.cpp

```diff
@@ -24,6 +24,22 @@ LogicalResult GENX::MatrixDPASOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// genx.matrix.2Dblockload
+//===----------------------------------------------------------------------===//
+
+LogicalResult GENX::Matrix2DBlockLoadOp::verify() {
+  if (getElemSizeInBits() != 8 && getElemSizeInBits() != 16 &&
```

Comment:
Improve the diagnostic to report why the element size is invalid and what it should be.